test/query_property_test.dart

```diff
@@ -149,7 +147,11 @@ void main() {
     query.close();
   });
 
-  final min = (a, b) => a < b ? a : b;
+  final intLt = (int a, int b) => a < b ? a : b;
```

Comment:
I find the `Lt` and `Gt` suffix confusing. Maybe `Min` and `Max` instead? Btw, is there no built-in min/max functionality?

Reply:
lol [there actually is](https://api.dart.dev/stable/2.10.4/dart-math/min.html)... didn't occur to me to doubt whoever wrote the original code... assumed they checked :shrug: 
removed those and used the dart:math lib